tools/nni_cmd/launcher.py

```diff
@@ -360,9 +360,13 @@ def launch_experiment(args, experiment_config, mode, config_file_name, experimen
         module_name = AdvisorModuleName.get(package_name)
     if package_name and module_name:
         try:
-            check_call([sys.executable, '-c', 'import %s'%(module_name)], stdout=PIPE, stderr=PIPE)
+            stdout_full_path, stderr_full_path = get_log_path(config_file_name)
+            with open(stdout_full_path, 'a+') as stdout_file, open(stderr_full_path, 'a+') as stderr_file:
+                check_call([sys.executable, '-c', 'import %s'%(module_name)], stdout=stdout_file, stderr=stderr_file)
         except CalledProcessError as e:
-            print_error('%s should be installed through \'nnictl package install --name %s\''%(package_name, package_name))
+            print_error('some errors happen when import package %s.' %(package_name))
+            print_log_content(config_file_name)
+            print_error('If %s is not installed, it should be installed through \'nnictl package install --name %s\''%(package_name, package_name))
```

Comment:
would be better to check whether this package needs installation then print this error message, because a package which does not need installation may also have import error.

Reply:
fixed.